src/snowflake/snowpark/functions.py

```diff
@@ -6567,7 +6567,7 @@ def to_binary(e: ColumnOrName, fmt: Optional[str] = None) -> Column:
     return builtin("to_binary")(c, fmt) if fmt else builtin("to_binary")(c)
 
 
-def to_array(e: ColumnOrName) -> Column:
+def to_array(*cols: Union[ColumnOrName, List[ColumnOrName]]) -> Column:
```

Comment:
```suggestion
def to_array(*e: Union[ColumnOrName, List[ColumnOrName]]) -> Column:
```
let's just still call it `e` as there could be someone calling it with `to_array(e=...)` and we don't want to break them